stock_owner_restriction/readme/USAGE.rst

```diff
@@ -0,0 +1,9 @@
+To use this module you need to:
+
+#. Go to *Inventory > Overview*
+#. Create a incoming delivery order.
```

Comment:
I think this is not limited to that kind of operation.

But that leads to another thought, shouldn't be good feature to set a field on picking type to enable the owner check or not ?

IMHO, I would at least add the use case in description.rst.